### ISSUE TITLE
enable bbox

### ISSUE THREAD

**Issue by Haikuch** [tyrasd/taghistory#13]
would love to see the tag history within a specific area

**Comment by tyrasd** [comment]
Thanks for the suggestion, but I have to admit that this would actually be a much harder task than what this tool currently can perform: It takes a full-history dump and does a preprocessing step to make the taghistory queries fast. This step would have to be repeated for each custom area/bbox after cutting out the respective region from the history-planet using a tool like [osm-history-splitter](https://github.com/MaZderMind/osm-history-splitter). Using that approach, one could set up a custom taghistory instance e.g. for a *specific country*. But this approach wouldn't at all scale to really custom bboxes.

Some ideas that may worth investigating:
* make many smaller taghistory databases for (for example) all z10 tiles -> aggregate custom bboxes and/or lower zoom levels by combining data from those tiles 
* create a (probably overpass-api like) backend database containing all object history geometries and perform taghistory-queries on that one

**Comment by Haikuch** [comment]
Yes I already imagined that this would be hard to work out. But preprocessing the countries would help a lot already. Maybe I will try to set up something for some countries.

**Comment by tyrasd** [comment]
> Maybe I will try to set up something for some countries.

Cool. I have to admit that the code in https://github.com/tyrasd/taghistory/tree/master/backend still lacks documentation and makefiles / build scripts. I'll get back to you as soon as I find the time to finish it up!

**Comment by smartm0use** [comment]
Hi, what about get a specific country OSM database and then run your application on it?

**Comment by tyrasd** [comment]
Sorry for the long silence here.

> create a (probably overpass-api like) backend database containing all object history geometries and perform taghistory-queries on that one

Me and my colleagues at @giscience have been (and are still) busy working on a solution for this. Take a look at https://ohsome.org/apps/dashboard/ (an API is also available at https://api.ohsome.org). Parts of the code are already open source (https://github.com/giscience/oshdb, more information about that are in https://doi.org/10.1186/s40965-019-0061-3), the rest will follow in the near future. I'll keep you updated.

**Comment by Klumbumbus** [comment]
>Take a look at https://ohsome.org/apps/dashboard/

Wow, this looks great!

**Comment by smartm0use** [comment]
Great, thank you very much!
Keep up the good work ;)
